src/commands/EntityCreateCommand.ts

```diff
@@ -51,13 +51,9 @@ export class EntityCreateCommand implements yargs.CommandModule {
                 } catch (err) { }
             }
 
-            const path = process.cwd() + "/" + (directory ? (directory + "/") : "") + filename;
-            const fileExists = await CommandUtils.fileExists(path);
-            if (fileExists) {
-                throw `File ${chalk.blue(path)} already exists`;
-            }
+              const path = (directory.startsWith("/") ? "" : process.cwd() + "/") + (directory ? (directory + "/") : "") + filename;
```

Comment:
It kinda looks to me that you're trying to handle absolute paths.

In that case, perhaps look at `path.resolve`?  Something like `path.resolve(directory, filename)` would get you probably much of the behavior to handle both relative and absolute paths.

Reply:
I just copy code from MigrationCreateCommand...
Maybe delegate task create file =T